src/interfaces/store-interfaces/state-interfaces/businessInformation-interface.ts

```diff
@@ -7,4 +7,6 @@ export interface BusinessInformationIF {
   legalName?: string
   foundingDate?: string // actually date-time
   hasRestrictions?: boolean
+  naicsCode?: string
+  naicsDescription?: string
```

Comment:
Why not put these in an object? Then you can do `getCurrentNaics` and `setCurrentNaics` instead of having to parse/construct the object every time you use it?

Reply:
I teetered on this one a bit. This was to keep it in sync with our snapshot of the business which houses these properties in the root of the `business` object, not nested.
Since Edit UI is fundamentally built off of that snapshot (not a filing, where it does become nested at the point of filing/saving) i landed on this. 

I don't think there is a right or wrong way here but that's how i ended up on this approach.

Reply:
I agree with your judgement.